code/modules/mob/living/combat/accuracy_checks.dm

```diff
@@ -29,7 +29,7 @@
 		if(used_intent.blade_class == BCLASS_STAB)
 			chance2hit += 10
 		if(used_intent.blade_class == BCLASS_PEEL)
-			chance2hit += 25
+			chance2hit += 15
```

Comment:
This affects all peel intents. The PR makes it seem like this is intended for longer weapons only. You can add an additional length check if you want the bonus to be lesser for those.